build.gradle

```diff
@@ -1,104 +1,47 @@
-import org.asciidoctor.gradle.jvm.AsciidoctorTask
 import java.text.SimpleDateFormat
+import java.util.*
 
 plugins {
-    id "org.asciidoctor.jvm.base" version "3.3.2"
-    id "org.asciidoctor.jvm.convert" version "3.3.2"
-    id "org.asciidoctor.jvm.pdf" version "3.3.2"
+    id "java"
+    id "application"
 }
+
+def group = "org.isaqb"
+def releaseVersion = System.getenv("RELEASE_VERSION")
+def localVersion = "LocalBuild"
+project.version = releaseVersion == null ? localVersion : releaseVersion
+
 repositories {
     mavenCentral()
 }
 
-asciidoctorj {
-    version = '2.5.3'
-}
-
-ext {
-    today = new Date()
-    versionDate = new SimpleDateFormat("yyyyMMdd").format(today)
-
-    releaseVersion = System.getenv("RELEASE_VERSION")
-    localVersion = "LocalBuild"
-    project.version = releaseVersion == null ? localVersion : releaseVersion;
-
-    curriculumFileName = "curriculum-template"
-    addSuffixToCurriculum = { suffix ->
-        for (extension in ["html", "pdf"]) {
-            File source = new File("${buildDir}/${curriculumFileName}.${extension}")
-            File target = new File("${buildDir}/${curriculumFileName}${suffix}.${extension}")
-
-            source.renameTo(target)
-        }
-    }
+dependencies {
+    implementation("org.asciidoctor:asciidoctorj:2.5.10")
+    implementation("org.asciidoctor:asciidoctorj-pdf:2.3.7")
 }
 
+def curriculumFileName = "curriculum-template"
```

Comment:
Können wir diese Sachen zu den anderen Definitionen werfen? Dann ist das nicht so zerrüttet.

Reply:
Du möchtest, dass diese drei defs nach Java zu den Konstanten kommen, habe ich das richtig verstanden?

Reply:
Nein, zu den paar Defs weiter oben :)

Reply:
Also in Zeile 13ff